pyproject.toml

```diff
@@ -3,7 +3,6 @@ name = "keep"
 version = "0.29.5"
 description = "Alerting. for developers, by developers."
 authors = ["Keep Alerting LTD"]
-readme = "README.md"
```

Comment:
Intentionally?

Reply:
yes <3 not using pypi atm and it requires the readme to be in the dockerfile